### PR TITLE
Kanvi/momentum_issue

### PR DESCRIPTION
**Pull request by kanvi-nervana**

### CODE REVIEW

File: test/test_momentum_op.cpp [hunk 1]
```diff
@@ -96,7 +97,9 @@ TEST(ReplaceModifier, Momentum1) {
   std::vector<tensorflow::Tensor> tf_outputs1;
   std::vector<tensorflow::Tensor> tf_outputs2;
   std::vector<tensorflow::Tensor> tf_outputs3;
+
   ASSERT_OK(tf_session.Run({{var_assign, accum_assign}}, &tf_outputs1));
+  std::vector<string> tf_outputs1_s = ConvertToString(tf_outputs1, 10);
```
Comment (shresthamalik) [inline test/test_momentum_op.cpp]: change the 10 that is hard coded to something that is computed.

Comment (kanvi-nervana) [reply test/test_momentum_op.cpp]: done

File: test/python/test_variableops_static_input.py [hunk 2]
```diff
@@ -36,6 +36,7 @@
 
 import ngraph_bridge
 import os
+from os import environ
```
Comment (kanvi-nervana) [inline test/python/test_variableops_static_input.py]: remove this

File: test/ci/buildkite/ngtf-cpu_ubuntu-variables.yaml [hunk 3]
```diff
@@ -66,7 +66,8 @@
   - command: |
       source /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv/bin/activate 
       pip install pytest
-      PYTHONPATH=`pwd`:`pwd`/tools:`pwd`/examples/mnist python3 test/ci/buildkite/test_runner.py \
```
Comment (shresthamalik) [inline test/ci/buildkite/ngtf-cpu_ubuntu-variables.yaml]: Add to resnet also

File: test/graph_rewrites/test_ng_var_update_ng_tensor.cc [hunk 4]
```diff
@@ -43,10 +43,6 @@ namespace testing {
 class NGVarUpdateNGTensorOpTest : public tensorflow::OpsTestBase {};
 
 TEST_F(NGVarUpdateNGTensorOpTest, KernelTest) {
-  list<string> env_vars{"NGRAPH_TF_NGVARIABLE_BUFFER_SHARING"};
```
Comment (shresthamalik) [inline test/graph_rewrites/test_ng_var_update_ng_tensor.cc]: do we run this test only for var build?

Comment (kanvi-nervana) [reply test/graph_rewrites/test_ng_var_update_ng_tensor.cc]: yes

**Pull request merged**